tests/manager/api/controller/test_opds_feed.py

```diff
@@ -419,7 +419,9 @@ def test_navigation(self, circulation_fixture: CirculationControllerFixture):
 
         library = circulation_fixture.db.default_library()
         lane = circulation_fixture.manager.top_level_lanes[library.id]
-        lane = circulation_fixture.db.session.merge(lane)
+        session = circulation_fixture.db.session
+        lane = session.merge(lane)
+        session.expire_all()
```

Comment:
This update was necessary to get the tests to pass in the ci environment.  I was seeing a detached instance failure.